tripleblindmarket/private-kit
# Fix Enable Location Popup Bug

#### Description:
Fix a bug on IOS where the user if constantly prompted to enable their location if they set their Location Permission to `Ask Next Time`. 

#### Linked issues:
https://pathcheck.atlassian.net/browse/SAF-731

#### How to test:

1. Install the app from scratch and go through onboarding. Select `Maybe Later` on the Location Permissions screen. You should not see the Enable Location popup on the next screen.

1. With the app already installed, set the Location Permissions to `Ask Next Time`. Close and reopen the app. You should not You should not see the Enable Location popup on the next screen.
